src/content/smart-bulletins/038.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Smart Bulletin No. 038"
+intro: "Notice Regarding Unenforceability of Unauthorized Obligations for Task Orders Awarded Under GSA SmartPay® 3 (SP3)"
+number: "No. 038"
+order: 038
+businessline: "Purchase, Travel, Fleet, Integrated"
+published: "Sep 26, 2023"
+lastupdated: "N/A"
+---
+
+UPDATE:
+| Date | Version History Action Log - Summary of Action/Changes |
+| ----------- | ----------- |
+| September 26, 2023 | Implementation of Smart Bulletin #38 |
+
+## Business Line(s) Affected
+
+ Purchase, Travel, Fleet, Integrated
+
+## Introduction
+
+The purpose of this Smart Bulletin is to provide clarity for customer agencies with
+task orders awarded under the SP3 master contracts. If any government
+authorized end user clicks on an "I agree" click box or other comparable
+mechanisms within the servicing bank system(s) this action may not automatically
+bind the government to the agreement. This includes any "I agree" or similar text
+that might be found in card management systems, including, but not limited to the
+Electronic Access Systems of the banks (US Bank and Citibank) and other
+systems such as MasterCard's Insights on Demand (IOD) system and Visa's
+Intellilink system
+
+## Summary
+
+The Department of Defense (DoD), General Service Administration (GSA), and the National Aeronautics and Space Administration (NASA) published an interim rule in the Federal Register at[78 FR 37686](https://www.federalregister.gov/citation/78-FR-37686) on June 21, 2013, to implement
```

Comment:
The Department of Defense (DoD), General Service Administration (GSA), and the National Aeronautics and Space Administration (NASA) published an interim rule in the Federal Register at[78 FR 37686](https://www.federalregister.gov/citation/78-FR-37686) on June 21, 2013, to implement the Department of Justice (DOJ) Office of Legal Counsel (OLC) opinion, entitled “Memorandum for Barbara S. Fredericks, Assistant General Counsel for Administration, United States Department of Commerce,” which opined that the Anti-Deficiency Act (ADA) ([31 U.S.C. 1341](https://api.fdsys.gov/link?collection=uscode&title=31&year=mostrecent&section=1341&type=usc&link-type=html)) is violated when a Government contracting officer or other employee with authority to bind the Government (emphasis added) agrees, without statutory authorization or other exception, to an open-ended, unrestricted indemnification clause.
On April 4, 2013, the Office of Management and Budget (OMB) issued guidance outlining a series of management actions to ensure agencies act in compliance with the ADA and in accordance with OLC’s opinion. This guidance clarifies that the inclusion of an open-ended indemnification clause in an End User License Agreement (EULA), online Terms of Service (TOS), or other agreement is not binding on the Government unless expressly authorized by statute and specifically authorized under applicable agency regulations and procedures, and shall be deemed to be stricken from the EULA, TOS, or similar legal instrument or agreement.
The GSA SmartPay 3 Master Contract includes Federal Acquisition Regulation (FAR) Clause 52.212-4, entitled “Contract Terms and Conditions - Commercial Items.” In addition, GSA SmartPay Master Contract Modification number 19 added GSA Acquisition Manual (GSAM) clause 552.232-39, entitled “Unenforceability of Unauthorized Obligations (FEB 2018) (DEVIATION FAR 52.232-39)” and GSAM Clause 552.232-78, “Commercial Supplier Agreements–Unenforceable.”
These GSA SmartPay 3 Master Contract clauses address situations when there is an unrestricted, open-ended indemnification provision in EULA, TOS, or similar legal instruments or agreements. The clauses clarify that if an EULA, TOS, or similar legal instrument or agreement includes a requirement for the Government to indemnify the contractor or any person or entity for damages, costs, fees, or any other loss or liability that would create an Anti-Deficiency Act violation, such requirements are unenforceable against the Government, and are deemed to be stricken from the agreement to prevent violations of the Anti-Deficiency Act.
It is important to note that the GSA SmartPay 3 Master Contract clauses referenced above are not intended to restrict the contractor bank from requiring government end users to sign account holder applications and agree to the submission of applications and/or account holder agreements. Government end users are still required to agree to the terms of these documents so long as they comply with the GSA SmartPay 3 master contracts and do not create Anti-Deficiency violations.
The following links provide additional information on applicable clauses and guidance pertaining to the Unenforceability of Unauthorized Obligations:
GSAM Clause 552.232-39, “Unenforceability of Unauthorized Obligations”:https://www.acquisition.gov/gsam/552.232-39
GSAM Clause 552.232-78, “Commercial Supplier Agreements–Unenforceable”:[https://www.acquisition.gov/gsam/552.232-78](https://www.acquisition.gov/gsam/552.232-39)